src/test/java/io/stargate/sgv2/jsonapi/api/v1/tables/InsertOneTableIntegrationTest.java

```diff
@@ -1550,18 +1552,18 @@ void insertValidVectorValueUsingBase64() {
 
       String inputJSON =
               """
-                          { "id": "vectorValidBase64",
```

Comment:
Not sure why `mvn fmt:format` decided to re-indent. Not needed or desired but... happened :-(